src/app/components/Input/InputRange.js

```diff
@@ -0,0 +1,21 @@
+const InputRange = ({ name, value, onChange }) => {
```

Comment:
Please move it to be in a components folder in your shadow box folder

Reply:
What if another developer wants to use this component? I think It would be good for not duplicating the code. I have also used the footer from the gradient generator and created it in the components folder. What do you think about this?

Reply:
> What if another developer wants to use this component? I think It would be good for not duplicating the code. I have also used the footer from the gradient generator and created it in the components folder. What do you think about this?

IDK, sure.